### PR TITLE
Bug 1215703 - Imagecompare RTL: increase accessibility menu coverage

### PR DESCRIPTION
**Pull request by nojunpark**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4ba4fc1c24b3d71cacdbf32ceae4136327a18789

File: tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/accessibility.py [hunk 1]
```diff
@@ -108,3 +120,39 @@ def a11y_toggle_invert(self):
 
     def a11y_toggle_grayscale(self):
         self.a11y_toggle_switch(self.marionette.find_element(*self._grayscale_switch_locator))
+
+
+class AccessibilityAudio(Base):
```
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/accessibility.py]: I think we should inherit from PageRegion here, normally.

File: tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/accessibility.py [hunk 2]
```diff
@@ -108,3 +120,39 @@ def a11y_toggle_invert(self):
...
+
+    _page_locator = (By.ID, 'accessibility-audio')
+
+    @property
+    def screen_element(self):
+        return self.marionette.find_element(*self._page_locator)
+
+
+class AccessibilityInput(Base):
```
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/accessibility.py]: This one should also inherit from PageRegion here, normally.

File: tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/accessibility.py [hunk 3]
```diff
@@ -108,3 +120,39 @@ def a11y_toggle_invert(self):
...
+        self.marionette.switch_to_frame()
+        Wait(self.marionette).until(expected.element_displayed(*self._confirm_delay_change_locator))
+
+    def tap_confirm_delay(self):
+        element = Wait(self.marionette).until(
+            expected.element_present(*self._confirm_delay_change_locator))
+        Wait(self.marionette).until(expected.element_displayed(element))
+        element.tap()
+        self.apps.switch_to_displayed_app()
+        Wait(self.marionette).until(expected.element_displayed(*self._delay_locator))
```
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/accessibility.py]: No newline at end of file.

File: tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/accessibility.py [hunk 4]
```diff
@@ -108,3 +120,39 @@ def a11y_toggle_invert(self):
...
+
+    def tap_change_delay(self):
+        element = Wait(self.marionette).until(
+            expected.element_present(*self._delay_locator))
+        Wait(self.marionette).until(expected.element_displayed(element))
+        element.tap()
+        self.marionette.switch_to_frame()
+        Wait(self.marionette).until(expected.element_displayed(*self._confirm_delay_change_locator))
+
+    def tap_confirm_delay(self):
```
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/accessibility.py]: Perhaps this should be part of a ConfirmDialog class?

Comment (npark-mozilla) [inline tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/accessibility.py]: At the moment I only take screenshots and do not do actual selection - once we expand the class to pick selection, we should put it in its own class.  I wonder whether there is a prototype class that we can create to use for system dialogs like this.

File: tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/accessibility.py (hunk 3 above)
Comment (npark-mozilla) [inline tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/accessibility.py]: changed

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8e076f0f74f46d1805d1981cceda7659d9f8ed6c

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=08a96ccc25c9817215ebc0ae1a1cbeb4620a139c

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=020b68fbf47cdeb33f42664fedd1042aa7710f42

**Pull request merged**